addons/binding/org.openhab.binding.autelis/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,568 @@
+		<description>The current state of feature/macro 3</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="equipment-feature4">
+		<item-type>Number</item-type>
+		<label>Feature 4</label>
+		<description>The current state of feature/macro 4</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="equipment-feature5">
+		<item-type>Number</item-type>
+		<label>Feature 5</label>
+		<description>The current state of feature/macro 5</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="equipment-feature6">
+		<item-type>Number</item-type>
+		<label>Feature 6</label>
+		<description>The current state of feature/macro 6</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="equipment-feature7">
+		<item-type>Number</item-type>
+		<label>Feature 7</label>
+		<description>The current state of feature/macro 7</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="equipment-feature8">
+		<item-type>Number</item-type>
+		<label>Feature 8</label>
+		<description>The current state of feature/macro 8</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="equipment-feature9">
+		<item-type>Number</item-type>
+		<label>Feature 9</label>
+		<description>The current state of feature/macro 9</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="equipment-feature10">
+		<item-type>Number</item-type>
+		<label>Feature 10</label>
+		<description>The current state of feature/macro 10</description>
+		<state readOnly="false">
+		</state>
+	</channel-type>
+	<channel-type id="temp-poolht">
+		<item-type>Number</item-type>
+		<label>Pool Heater Status</label>
+		<description>The current pool/hi-temp heater settings</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Off</option>
+				<option value="1">Heater</option>
+				<option value="2">Solar Preferred</option>
+				<option value="3">Solar Only</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="temp-spaht">
+		<item-type>Switch</item-type>
+		<label>The current spa/lo-temp heater settings</label>
+		<description>Spa Heat</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Off</option>
+				<option value="1">Heater</option>
+				<option value="2">Solar Preferred</option>
+				<option value="3">Solar Only</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="temp-htstatus">
+		<item-type>Number</item-type>
+		<label>Heat Status</label>
+		<description>Heat Status </description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="temp-poolsp">
+		<item-type>Number</item-type>
+		<label>Pool Heat Setpoint</label>
+		<description>The current pool/hi-temp setpoint</description>
+	</channel-type>
+	<channel-type id="temp-spasp">
```

Comment:
How do you think this will this work if it supports both C and F? The ranges, step and pattern will be different.